flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/KeyedProcessOperatorTest.java

```diff
@@ -397,17 +396,16 @@ public void processElement(Integer value, Context ctx, Collector<String> out) th
 		}
 
 		@Override
-		public void onTimer(
-				long timestamp,
-				OnTimerContext ctx,
-				Collector<String> out) throws Exception {
+		public void onTimer(long timestamp, OnTimerContext ctx, Collector<String> out) throws Exception {
 		}
 	}
 
 	private static class TriggeringFlatMapFunction extends ProcessFunction<Integer, Integer> {
 
 		private static final long serialVersionUID = 1L;
 
+		static final int TEST_VALUE = 17;
+
 		private final TimeDomain timeDomain;
 
 		public TriggeringFlatMapFunction(TimeDomain timeDomain) {
```

Comment:
rename `timeDomain` -> `expectedTimeDomain` and add `expectedKey` and add assertion for the expected key to `onTimer()` method triggered both in `KeyedProcessOperatorTest#testEventTimeTimers` and `KeyedProcessOperatorTest#testProcessingTimeTimers`